code/modules/research/experimentor/loot/device_exploot.dm

```diff
@@ -0,0 +1,236 @@
+//////////////////////////////DEVICE DEFINITION////////////////////////////////
+// Re-implements the old strange object code in order to finish the proof of concept.
+// Loot and name lists are defined in _loot_definer.dm for each object type.
+
+#define RARITY_COMMON 0
+#define RARITY_UNCOMMON 1
+#define RARITY_RARE 2
+#define RARITY_VERYRARE 3
+
+/obj/item/discovered_tech
+	name = "Discovered Technology"
+	desc = "A strange device. Its function is not immediately apparent."
+	icon = 'icons/obj/assemblies.dmi'
+	origin_tech = "combat=1;plasmatech=1;powerstorage=1;materials=1"
+	var/list/iconlist = list("shock_kit","armor-igniter-analyzer","infra-igniter0","infra-igniter1","radio-multitool","prox-radio1","radio-radio","timer-multitool0","radio-igniter-tank")
+	var/cooldownMax = 60
+	var/cooldown = FALSE
+	var/stability = 0
+	var/potency = 0
+	var/raritylevel = 0
+	var/base_name = "Unknown"
+	var/extra_data
+	var/box_type
+
+	var/list/keywords = list("clowns")
+	var/use_generated_names = TRUE
+	var/use_generated_descriptions = TRUE
+
+	var/techLevels
+
+/obj/item/discovered_tech/New(var/stability_in, var/potency_in, var/rare_level, var/boxtype)
+	stability = stability_in
+	potency = potency_in
+	raritylevel = rare_level
+	icon_state = pick(iconlist)
+	techLevels = raritylevel*2
+	box_type = boxtype
+	initialize()
+
+/obj/item/discovered_tech/attack_self(mob/user)
+	if(cooldown)
+		to_chat(user, "<span class='warning'>The [src] does not react!</span>")
+		return
+	else if(src.loc == user)
+		cooldown = TRUE
+		itemproc(user)
+		spawn(cooldownMax)
+			cooldown = FALSE
+
+/obj/item/discovered_tech/proc/initialize()
+	cooldownMax = round(rand(150,300)*(1.4-0.2*raritylevel)*(1.4-0.2*(stability/20)),1)
+	origin_tech = ""
+	if(box_type == "Mysterious")
+		origin_tech += "syndicate=[1+raritylevel+rand(raritylevel,raritylevel+1)];"
+	if(box_type == "Alien")
+		origin_tech += "abductor=[1+raritylevel+rand(raritylevel,raritylevel+1)];"
+
+/obj/item/discovered_tech/proc/itemproc(mob/user)
+	return
+
+/obj/item/discovered_tech/proc/warn_admins(mob/user, ItemType, priority = 1)
+	var/turf/T = get_turf(src)
+	var/log_msg = "[ItemType] experimentor tech used by [key_name(user)] in ([T.x],[T.y],[T.z])"
+	if(priority) //For truly dangerous relics that may need an admin's attention. BWOINK!
+		message_admins("[ItemType] experimentor tech activated by [key_name_admin(user)] in ([T.x], [T.y], [T.z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[T.x];Y=[T.y];Z=[T.z]'>JMP</a>)",0,1)
```

Comment:
use `ADMIN_COORDJMP` instead of the manual typing

Reply:
message_admins will probably also be to much for logging. Admins will have to give you pointers which logging is better to use though